### PR TITLE
feat(stability-api): make it optional

### PR DESCRIPTION
**Pull request by badayvedat**

### CODE REVIEW

File: py/libs/stability.py [hunk 1]
```diff
@@ -152,7 +152,10 @@ async def getUserBalance(self):
             PromptServer.instance.send_sync('easyuse-toast', {'type': 'error', 'content': self.getErrors(response.status_code)})
             return None
 
-stableAPI = StabilityAPI()
+if os.environ.get("COMFYUI_EASY_USE_DISABLE_STABILITY_API", "0") == "1":
+    print("Stable Diffusion 3 API is not enabled")
+else:
+    stableAPI = StabilityAPI()
```
Comment (badayvedat) [inline py/libs/stability.py:158]: if this flag is enabled it wont be enabled at all

**Comment by yolain** [comment]
I don't quite understand the need to make it optional. However，this node is outdated, I actually want to set it to deprecated.

**Comment by badayvedat** [comment]
for cases where the comfyui is shared and the API key does not exist
https://github.com/comfyanonymous/ComfyUI/blob/15c39ea75774b5d36065eb8caa5ecbbfcd168b71/comfy/cli_args.py#L139
but yeah might be an overkill, lets not over-complicate it

and after you deprecate it, do you plan to add a new node that will replace it ? 👀

**Comment by yolain** [comment]
I think the usage is very low, and the node was created in the first place because STABILITY doesn't publicize model weights unless followed by a very shiny API capability.

**Pull request closed without merging**